src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const PropertyAndValue = sourceString.split(';');
+  const stileList = {};
+  const property = [];
+  const value = [];
 
+  for (let i = 0; i < PropertyAndValue.length; i++) {
+    if (i === 26) {
+      return stileList;
+    }
+    property[i] = PropertyAndValue[i].slice(3,
+      (PropertyAndValue[i].indexOf(':')));
```

Comment:
why do you place this on the new line?

Reply:
because linter swears at the length of the string

Reply:
<img width="619" alt="Screen Shot 2019-08-19 at 1 44 21 PM" src="https://user-images.githubusercontent.com/36257947/63259575-791fa000-c287-11e9-941e-29b97d11688b.png">

Reply:
Oh, that... Ok. Fix.